internal/clients/service.go

```diff
@@ -1,16 +1,26 @@
 package clients
 
 import (
+	"crypto/tls"
+	"crypto/x509"
 	"encoding/json"
+	"fmt"
 	"os"
 
 	"golang.org/x/exp/slog"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
 
 	"go.temporal.io/sdk/client"
 )
 
 type TemporalServiceConfig struct {
 	HostPort string `json:"hostPort"`
+	UseTLS   bool   `json:"useTLS"`
+	CACert   string `json:"caCert"`
+	CertFile string `json:"certFile"`
```

Comment:
The property includes "file" which could be mixed up with a filepath.

What about:
```
CACertPem string `json:"caCertPem"`
CertPem string `json:"certPem"`
KeyPem  string `json:"keyPem"`
```
?